src/overlays/actors/ovl_En_Ossan/z_en_ossan.c

```diff
@@ -1342,12 +1342,12 @@ void EnOssan_InitPartTimeWorker(EnOssan* this, GlobalContext* globalCtx) {
 
 s32 EnOssan_GetWelcomeCuriosityShopMan(EnOssan* this, GlobalContext* globalCtx) {
```

Comment:
If we're returning a textId in line 1350, I don't think s32 is necessarily the right return type for this function. Do s16 or u16 match? (I am so sorry for making this PR about En_Ossan).

Reply:
Don't worry, if we can improve the decomped code, we should do it

Reply:
The change actually matches :+1:

Reply:
The change to what? u16?

Reply:
Yes, it works returning a u16

Reply:
judging by your latest commit, yes, u16. That's great news! Thanks for incorporating all my changes :)